app/Http/Controllers/DashboardController.php

```diff
@@ -36,4 +45,46 @@ private function getTopLoadouts()
 
         return $loadouts;
     }
+
+    private function getRecentTopLoadouts()
+    {
+        $loadouts = collect();
+        $characterIds = Character::pluck('id')->sortBy('id');
+
+        foreach ($characterIds as $characterId) {
+            $characterLoadouts = Loadout::where([
+                ['character_id', $characterId],
+                ['created_at', '>', Carbon::now()->subDays(14)], // Loadouts created in the past 2 weeks
+            ])
+                ->withCount('votes')
+                ->with('character', 'creator', 'mods.gun')
+                ->orderBy('votes_count', 'desc')
+                ->take(3)
+                ->get();
+            $loadouts->push($characterLoadouts);
+        }
+
+        return $loadouts;
+    }
+
+    private function getLatestLoadouts()
```

Comment:
Same for this, we should make it class agnostic imo. The only one we leave is maybe the top per character that we had before and that goes at the bottom.